android/keystores/debug.keystore.properties

```diff
@@ -1,4 +0,0 @@
-key.store=debug.keystore
```

Comment:
Not sure if this is related, but it looks like a debug key is needed for `react-native start`? I'm not able to build due to missing key `/mobile/android/app/debug.keystore` 😢 .

Reply:
can you paste the error output please?

Reply:
Snippet of `react-native run-android` output:

```
Execution failed for task ':app:validateSigningDebug'.
> Keystore file '/Users/williamthomson/Projects/sussol/mobile/android/app/debug.keystore' not found for signing config 'debug'.
```

Reply:
I'm building fine :( If you don't mind would you please try adding this back to see if it makes a difference to you as a first step? Sorry to put that on you! I'm gonna look into keys now as I am unsure about them (so dont go deleting stuff you dont know about?)

Reply:
Not sure of the cause, have reverted a few things and still seems to occur. Have fixed by building an arbitrary key and using that:

```
keytool -genkey -v -keystore debug.keystore -storepass android -alias androiddebugkey -keypass android -keyalg RSA -keysize 2048 -validity 10000
```

Will go ahead and resolve this.